nautobot/docs/administration/nautobot-server.md

```diff
@@ -272,6 +272,40 @@ $ nautobot-server nbshell
 
 Please see the dedicated guide on the [Nautobot Shell](nautobot-shell.md) for more information.
 
+### `populate_database`
+
++++ 1.5.0
+
+`nautobot-server populate_database [--flush] --seed SEED`
+
+Populate the database with various data as a baseline for testing (automated or manual).
+
+`--flush`  
+Flush any existing data from the database before generating new data.
+
+!!! danger
+    Running this command with the `--flush` argument will clear all existing data in your database. You have been warned.
```

Comment:
There is no mercy in this dojo!!!!

Reply:
I should probably add something similar to https://github.com/django/django/blob/main/django/core/management/commands/flush.py#L59, i.e. a confirmation prompt if someone runs this command interactively?